### PR TITLE
Copyedit facial-recognition-ai-python

### PR DESCRIPTION
**Pull request by charlottejmc**
I've now prepared my copyedits for `/en/facial-recognition-ai-python` represented in Issue #552.

### CODE REVIEW

**Comment by charlottejmc** [comment]
Hello @c-goldberg and @giuliataurino. I've now prepared the copyedits for this lesson. I'd be grateful if you could review the adjustments and confirm that you are happy for me to merge these. You can see the details of my edits under the [files changed](https://github.com/programminghistorian/ph-submissions/pull/617/files) tab!

I'd like to bring your attention to the comments I have attached to specific lines. You can respond to any of my suggestions via the comments below, or click _Resolve conversation_ if you're happy with them like this:

image

If you want to make edits, please work [here](https://github.com/programminghistorian/ph-submissions/pull/617/files) by clicking the three dots at the upper right of the file, then _Edit file_:

image

File: en/drafts/originals/facial-recognition-ai-python.md [hunk 1]
```diff
@@ -415,37 +425,47 @@ data.to_csv('YearbookOutput.csv', index=False)
...
+If we draw some qualitative conclusions from this small sample dataset, 1961 is the first year where smiles began to rival non-smiles in total numbers. Claudia Goldin and Lawrence F. Katz have argued that while American higher education gradually became more and more coed (mixed genders) from the mid-1800s to the mid-1900s, the switch from single-sex to coed education accelerated rapidly in the 1960s and 1970s.[^9] The greater share of smiling faces in yearbooks from the 1960s might be a part of other, wider social changes in the American collegiate landscape. Further analysis of this data by sorting faces based on gender would be needed to confirm whether Bethel's student body became markedly more female in these years, but of course that process would be fraught by the ethical challenges of gender-based facial recognition [described above](#ethical-issues-with-facial-recognition).
 
-Let's continue our analysis and zero in on the "Error Weight" column of the `.csv` file:
+Let's continue our analysis and zero in on the **Error Weight** column of the `.csv` file:
```
Comment (charlottejmc) [inline en/drafts/originals/facial-recognition-ai-python.md:443]: As part of our commitment to keeping our pages as light and accessible as possible, we prefer to convert images of spreadsheets to markdown tables instead.

File: en/drafts/originals/facial-recognition-ai-python.md [hunk 2]
```diff
@@ -415,37 +425,47 @@ data.to_csv('YearbookOutput.csv', index=False)
...
-{% include figure.html filename="or-en-facial-recognition-ai-python-10.png" alt="Visual description of figure image" caption="Figure 10. Screenshot of `.csv` file showing experiment results" %}
+|   | A    | B            | C           | D           |
+|---|------|--------------|-------------|-------------|
+| 1 | Years| Smiles       | NonSmiles   | Error Weight|
+| 2 | 1911 | 0            | 1           | 0           |
+| 3 | 1921 | 0            | 0.923076923 | 0.076923    |
+| 4 | 1931 | 0            | 0.7         | 0.3         |
+| 5 | 1941 | 0.066666667  | 0.4         | 0.533333    |
+| 6 | 1951 | 0.171929825  | 0.684210526 | 0.14386     |
+| 7 | 1961 | 0.385964912  | 0.526315789 | 0.087719    |
```
Comment (charlottejmc) [inline en/drafts/originals/facial-recognition-ai-python.md:452]: As part of our commitment to keeping our pages as light and accessible as possible, we prefer to convert images of spreadsheets to markdown tables instead.

**Review by c-goldberg — APPROVED** [review]

**Comment by c-goldberg** [comment]
Thank you @charlottejmc! I think these all look great.

**Pull request merged**